priv/search-input.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
Let's make this a JS module so we don't have to turn this on manually.

Reply:
Script is now being loaded with `type="module"`.

Let me know if you also want to use the `.mjs` extension instead of just plain `.js`.